src/DBUtils/mysql_db_utils.jl

```diff
@@ -144,19 +150,29 @@ function create_server(con::MySQL.MySQLHandle, dbname; linkname = "fedlink", use
 end
 
 """
-    disable_foreing_checks(con::MySQL.MySQLHandle)
-Disables foreing checks for MySQL database
+    disable_foreign_checks(con::MySQL.MySQLHandle)
+Disables foreign checks for MySQL database
 """
-function disable_foreing_checks(conn::MySQL.MySQLHandle)
+function disable_foreign_checks(conn::MySQL.MySQLHandle)
     MySQL.execute!(conn, "SET FOREIGN_KEY_CHECKS = 0")
-
+    return nothing
 end
 
 """
-    enable_foreing_checks(con::MySQL.MySQLHandle)
-Enables foreing checks for MySQL database
+    enable_foreign_checks(con::MySQL.MySQLHandle)
+Enables foreign checks for MySQL database
 """
-function enable_foreing_checks(conn::MySQL.MySQLHandle)
+function enable_foreign_checks(conn::MySQL.MySQLHandle)
     MySQL.execute!(conn, "SET FOREIGN_KEY_CHECKS = 1")
+    return nothing
+end
 
-end
\ No newline at end of file
+"""
+    set_innodb_checks(conn, autocommit = 1, foreign_keys = 1, unique = 1)
+"""
+function set_innodb_checks!(conn::MySQL.Connection, autocommit::Int = 1, foreign_keys::Int = 1, unique::Int = 1)
+    MySQL.execute!(conn, "SET FOREIGN_KEY_CHECKS = $foreign_keys")
+    MySQL.execute!(conn, "SET AUTOCOMMIT = $autocommit")
+    MySQL.execute!(conn, "SET UNIQUE_CHECKS = $unique")
+    return nothing
+end
```

Comment:
with your new function, the old enable/disable are redundant and not needed

Reply:
these have different use cases/functionality so should remain different